README.md

```diff
@@ -2,25 +2,64 @@
 
 # operatorkit
 
-The operatorkit package implements an opinionated framework for developing
-Kubernetes operators. It can be used as a library in golang. It is built on
-top of our [microkit](https://github.com/giantswarm/microkit) framework which
-provides base functionality like logging and error handling.
+The operatorkit is a library for creating [Kubernetes operators][operators]. It
+emerged as extracted common functionality of number of the operators we
+developed in Giant Swarm. The goal of the library is to provide common
+structure of the operator projects and encapsulate best practices learned
+during running operators in production.
+
+## Features
+
+- Reducing boilerplate code.
+- CRD/TPR primitives allowing to reliably create, watch custom resources, and
```

Comment:
`primitives allow`

Reply:
Maybe `create, watch and delete`? I would maybe create another bullet point for the decoding part.

Reply:
I changed to

```
- CRD/TPR primitives to reliably create, watch and delete custom resources.
- Provides custom object decoder reducing boilerplate code to the minimum.
```

I don't know about CRD deleting functionality and I'm not sure how useful it is.

Reply:
Good enough for now.